### DISCUSSION TITLE
Index Planning

### DISCUSSION THREAD

**Discussion by StephanBerio** [witsa/synapps#775] [Q&A]
Bonjour,
j'ai un bouton de navigation qui affiche dans une modale le planning hebdomadaire
donc en paramètre de mon bouton se trouve l'aditionnel propPlanning qui donne le chemin du planing.
comment à partir de cet additionnel puis recupérer l'index du planning.
Mon but est de rendre le bouton vert (la couleur de fond) quand l'index du planning est à 1 et gris quand l'index est à 0.
Ca marche si je lie la couleur de fond à un fournisseur de variable pointant sur le planning, 
mais ça me fait renseigner 2 fois le même planning.
Merci pour votre aide. Bonne Journée. Cdt Stéphan

**Comment by WITNicolas** [comment]
Tu ne peux pas récupérer l'index sans utiliser un fournisseur de données.
Il te faut une additionnelle dans le Bouton que tu liaisonneras avec un planning, et ça passera obligatoirement par une source de données si tu veux réutiliser le chemin que tu renseigne déjà dans ton bouton

Dés lors que tu as un mécanisment "complexe", il faut faire des interactions entre acteurs et donc créer un composite à partir des acteurs de bases.

(Si j'ai bien compris :) )
